plugins/aggregators/derivative/README.md

```diff
@@ -0,0 +1,166 @@
+# Derivative Aggregator Plugin
+The Derivative Aggregator Plugin estimates the derivative for all fields of the
+aggregated metrics.
+
+### Time Derivatives
+
+In its default configuration it determines the first and last measurement of
+the period. From these measurements the time difference in seconds is
+calculated. This time difference is than used to divide the difference of each
+field using the following formula:
+```
+              field_last - field_first
+derivative = --------------------------
+                  time_difference
+```
+For each field the derivative is emitted with a naming pattern
+`<fieldname>_by_time`.
```

Comment:
how do you feel about _rate instead of _by_time?

Reply:
Actually the suffix is configurable. `_by_time` is currently the default value. I can change it to `_rate` if you prefer that.